docs/source/custom_directives.py

```diff
@@ -39,16 +39,26 @@ def run(self):
         header = self.options.get("header", "")
         description = self.options.get("description", "")
         link = self.options.get("link", "")
-        image = '<img src="%s">' % self.options.get("image", "")
+        image = self.options.get("image", "")
         tags = self.options.get("tags", "")
 
-        card_rst = _CUSTOM_CARD_TEMPLATE.format(
-            header=header,
-            description=description,
-            link=link,
-            image=image,
-            tags=tags,
-        )
+        if image:
```

Comment:
There was a bug here that prevented images from appearing on the tutorial/recipe cards that *do* have images. It's fixed now